test/unit_tests/apps/ccdb-landing-map/js/Chart-spec.js

```diff
@@ -0,0 +1,39 @@
+import Chart from '../../../../../cfgov/unprocessed/apps/ccdb-landing-map/js/Chart';
```

Comment:
Ugly as this is, I like that in modern Javascript it will throw an error if it moves.

It's another way of validating a contract, but in this case, a file location